### ISSUE TITLE
Request: could you also share script and a bit more descriptions on deepcoder_1.5b training?

### ISSUE THREAD

**Issue by insop** [agentica-project/rllm#83]
Hi Agentica team,

Thank you for the great work!

I was wondering if you could share a few more details about the training of `deepcoder_1.5b`?
In your blog, you shared result from `deepcoder_1.5b`, and I'm planning to run a small test run myself.

Here are a few things that would be especially helpful:
- `deepcoder_1.5b_coding_*k.sh` scripts
- number of training steps for each lengths, `8k`,`16k`, and `24k`
- the loss plot from the training of `deepcoder_1.5b`

Thank you!

**Comment by michaelzhiluo** [comment]
We only trained Deepcoder 1.5B coding on 32K context 😆 and let it run while we did other stuff.

The script for this will be updated shortly in the repo... the wandb logs for deepcoder 1.5b is already in wandb.ai/mluo/deepcoder

**Comment by insop** [comment]
> We only trained Deepcoder 1.5B coding on 32K context 😆 and let it run while we did other stuff.
> 
> The script for this will be updated shortly in the repo... the wandb logs for deepcoder 1.5b is already in wandb.ai/mluo/deepcoder

I see, thank you for the clarification.
Here is the [config](https://wandb.ai/mluo/deepcoder/runs/s3lpnxwa/overview) from wandb, in case others to see.
